sandbox/pages/StatusInputPage.qml

```diff
@@ -196,4 +196,12 @@ Column {
             toggled = !toggled
         }
     }
+
+    StatusInputWithCursor {
+        id: cursor
+        input.placeholderText: "0.00 SNT"
+        Component.onCompleted: {
+            cursor.input.edit.forceActiveFocus()
+        }
```

Comment:
We have an issue here https://github.com/status-im/StatusQ/issues/578 which describes that all inputs should have a blue cursor. This seems similar to what you've done, just that `StatusInputWithCursor` not only is a StatusInput with a cursor, it also looks entirely different.

So I'm wondering:

1. Should we name this differently because it doesn't look like a standard status input
2. Or, is there a way to add this cursor functionality to the basic input so that all benefit from that?

Reply:
Aaah I had no idea about this.

I guess I could rename it to a different name? and whoever implements https://github.com/status-im/StatusQ/issues/578 can move this from here to there?

Reply:
as discussed I will move this over to desktop and add a to-do there for when https://github.com/status-im/StatusQ/issues/578 is implemented to remove cursor implementation from desktop component.

Will also link my implementation of the cursor to #578  for reference